support/cas-server-support-oidc/src/main/java/org/apereo/cas/oidc/config/OidcConfiguration.java

```diff
@@ -187,6 +188,9 @@
 @EnableConfigurationProperties(CasConfigurationProperties.class)
 @Slf4j
 public class OidcConfiguration implements WebMvcConfigurer {
+
+    public static final String POST_LOGOUT_REDIRECTURL_MATCHER_BEAN_NAME = "postLogoutRedirectUrlMatcher";
```

Comment:
To keep consistent:

1. Move constant into the interface
2. Follow the `String BEAN_NAME_XYZ = "..."` pattern.

Reply:
which interface?

Reply:
`OidcPostLogoutRedirectUrlMatcher`